.changeset/twelve-rabbits-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+"@near-js/wallet-account": patch
```

Comment:
This PR adds new functionality, meaning that it should be a minor version release